INSTALL

```diff
@@ -3,10 +3,10 @@ Look at https://github.com/sblaisot/automatic_addressbook
 Installing automatic addressbook plugin
 ---------------------------------------
 
-You need roundcube > 0.4 installed.
+You need roundcube > 0.9 installed.
 
-automatic_addressbook plugin version 0.2 has been tester against roundcube 0.5 and 0.6.
-above versions have been tested against roundcube 0.8.0 and 0.8.1 with larry skin and mysql backend.
+automatic_addressbook plugin version 0.2 has been tester against roundcube 0.9
```

Comment:
tester -> tested.
Please do not update this one concerning automatic_addressbook v0.2 (an old version) or remove it completely

Reply:
as i said.. can you merged and later just updated by yourselft? can you explain me more in detail?